aspnetcore/mvc/views/tag-helpers/builtin/resources/CacheTagHelper.md

```diff
@@ -0,0 +1,286 @@
+﻿[Back To Built In Tag Helpers List](../../builtin.md)
+
+# Cache Tag Helper
+
+By [Peter Kellner](http://peterkellner.net) 
+
+
+The  Cache Tag Helper provides the ability to dramatically improve the performance of your ASP.NET Core app by caching its content to the internal ASP.NET Core cache provider.
+
+A simple example that shows the Cache Tag Helper in action sets the current time in its content area. The Razor View Engine sets the default `expires-after` to twenty minutes, which is the default if you specify no additional attributes.
+
+Here is an example of the code you include in your `cshtml` page to achieve the default cache behavior described above.
+
+```html
+<Cache>@DateTime.Now<Cache>
+```
+
+If you repeatedly refresh the page that contains `CacheTagHelper`, the first time the current`DateTime`will be shown. Repeated refreshes of the page will continue to show the first `DateTime` shown until the cache expires (which is likely to be twenty minutes unless server memory pressure evicts the cache sooner).
+
+You can achieve more control of the cache duration of the content by setting any of the following attributes:
+
+## Cache Tag Helper Attributes
+
+- - -
+
+### enabled    
+
+
```

Comment:
... and here ... and I'll stop repeating this comment, as it looks like these extra lines are common going forward.